compiler/qsc_passes/src/capabilitiesck/tests_common.rs

```diff
@@ -76,7 +85,8 @@ impl CompilationContext {
     }
 
     fn new_for_exe(source: &str) -> Self {
-        let (std_id, store) = qsc::compile::package_store_with_stdlib(TargetCapabilityFlags::all());
+        let mut store = qsc::PackageStore::new(qsc::compile::core());
```

Comment:
Is it possible this was a bad merge?

Reply:
To clarify, I can't tell from the diff why the utility function you introduced won't work here. Did something change or was this unintentional?